air/src/proof/mod.rs

```diff
@@ -404,3 +406,259 @@ pub fn ceil(value: f64) -> f64 {
 pub fn ceil(value: f64) -> f64 {
     libm::ceil(value)
 }
+
+#[cfg(test)]
+mod prove_security_tests {
+    use super::ProofOptions;
+    use crate::{proof::get_proven_security, FieldExtension};
+    use math::{fields::f64::BaseElement, StarkField};
+
+    #[test]
+    fn get_96_bits_security() {
+        let field_extension = FieldExtension::Cubic;
+        let base_field_bits = BaseElement::MODULUS_BITS;
+        let fri_folding_factor = 8;
+        let fri_remainder_max_degree = 127;
+        let grinding_factor = 20;
+        let mut blowup_factor = 4;
+        let mut num_queries = 80;
+        let collision_resistance = 128;
+        let trace_length = 1 << 18;
```

Comment:
minor nit: should we use `2_usize.pow(18)` to make the operation a bit more explicit?

Reply:
Done